### PR TITLE
Update EIP-4844: update to Review status

### PR DESCRIPTION
**Pull request by protolambda**
This PR updates the EIP to reflect the review status: the EIP is complete, targeted for EIP-4844 devnet-v3, and will be adjusted based on further external review and testnet results. Maybe long overdue, it's exactly 270 days (9 months) since the PR was first created, and a lot of progress has been made!

### CODE REVIEW

**Comment by protolambda** [comment]
EIP-5793, which is a soft-dependency of this EIP (the EIP will work without, but propagation of larger transactions is derisked with the functionality of https://eips.ethereum.org/EIPS/eip-5793 ), still has a draft status and thus blocks the Review status of 4844. I will reach out to Marius and discuss the status of that EIP.

**Comment by protolambda** [comment]
Blocked by:
- `eth/68` status change: https://github.com/ethereum/EIPs/pull/6031
- `eth/67` status change: https://github.com/ethereum/EIPs/pull/6032

**Comment by protolambda** [comment]
>          Does this still block this PR?

Not that PR specifically, but that EIP should still move to Review status (if not just Final already, since it has been adopted).

The status change of 4844 is blocked by:
- EIP-1 accepting devp2p repo links: https://github.com/ethereum/EIPs/pull/6034
- EIP-4938 first moving from Stagnant to Draft: https://github.com/ethereum/EIPs/pull/6014
- EIP-4938 then moving from Draft to Review in separate PR
  - And it should probably be moved further than Review before it becomes Stagnant again, it's been adopted already.
- EIP-5793 moving to Review: https://github.com/ethereum/EIPs/pull/6031

**Comment by Pandapip1** [comment]
@protolambda would you mind rebasing? EIPW is giving errors still

File: EIPS/eip-4844.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Shard Blob Transactions
 description: Shard Blob Transactions scale data-availability of Ethereum in a simple, forwards-compatible manner.
 author: Vitalik Buterin (@vbuterin), Dankrad Feist (@dankrad), Diederik Loerakker (@protolambda), George Kadianakis (@asn-d6), Matt Garnett (@lightclient), Mofi Taiwo (@Inphi), Ansgar Dietrichs (@adietrichs)
 discussions-to: https://ethereum-magicians.org/t/eip-4844-shard-blob-transactions/8430
-status: Draft
+status: Review
 type: Standards Track
```
Comment (Erik4569) [inline EIPS/eip-4844.md:8]: hi

**Review by Erik4569 — APPROVED** [review]

**Review by Erik4569 — APPROVED** [review]

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**